students/synowiec_krzysztof/lesson_01_basics/helloHarry.py

```diff
@@ -0,0 +1,7 @@
+
+def main():
+    name = input()
+    print("Hello, " + name)
+
+if __name__ == '__main__':
+    main()
```

Comment:
Exclamation mark at the end is missing!

Please check code at snakify if it comply with all requirements and if it passes all tests:
https://snakify.org/lessons/print_input_numbers/problems/hello_harry/